tasks/task02/triangle_area.py

```diff
@@ -1,53 +1,72 @@
 import math, re
 from textwrap import dedent
 
-sel  = ''
-area = 0.0
-
 class clr:
     YELLOW  = '\033[38;5;193m'
     ORANGE  = '\033[38;5;209m'
     ENDC    = '\033[0;0m'
 
 def mainmenu():
-    sel = input(dedent(f'''
-    Menu:
-    1. Calculate triangle area by base {clr.ORANGE}and{clr.ENDC} height
-    2. Calculate triangle area by 2 sides {clr.ORANGE}and{clr.ENDC} angle between them
-    3. Exit
-    Enter menu item number: '''))
-
-    if sel not in ['1', '2', '3']:
+    m  = ''
+    while m not in ['1', '2', '3']:
+        m = input(dedent(f'''
+        Menu:
+        1. Calculate triangle area by base {clr.ORANGE}and{clr.ENDC} height
+        2. Calculate triangle area by 2 sides {clr.ORANGE}and{clr.ENDC} angle between them
+        3. Exit
+        Enter menu item number: '''))
+        if m in ['1', '2', '3']:
```

Comment:
So you repeated the same condition "m in ['1', '2', '3']". Is is a good sign that one of the conditions is excessive :)

Try to refactor this function to remove code duplication